Additional info about Dynamic entry (#2641)
webpack/webpack.js.org: src/content/configuration/entry-context.md

### Dynamic entry

> If a function is passed then it will be invoked on every [make](https://webpack.js.org/api/compiler-hooks/#make) event.

Reviewer: Please use relative URLs: 
```suggestion
If a function is passed then it will be invoked on every [make](/api/compiler-hooks/#make) event.
```

Author: Thanks

> > Note that the `make` event triggers when webpack starts and for every invalidation in watch-mode.

Reviewer 2: This is contradicting to other mode option in webpack configuration. Let's reword. Also make event shouldn't be quoted for consistency with first occurrence 
```suggestion
> Note that the make event triggers when webpack starts and for every invalidation when [watching for file changes](/configuration/watch/).
```

};
```

> > For example: you can use dynamic entries to get the actual entries from an external source (remote server, file system content or database)

Reviewer 2: If we are going to provide an example, don't put it into a note block:
```suggestion
For example: you can use dynamic entries to get the actual entries from an external source (remote server, file system content or database)
```

Also when we provide any example context, code block should follow. Please add abstract example in format of:

__webpack.config.js__

```javascript
// webpack config with abstract example corresponding to the given context
```

Author: fixed, thanks
